docs/changes/io.ascii/16780.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add functionality to read and write to a Table from the TDAT format as part of
+the Unified File Read/Write Interface and tests for basic functionality.
+Fixes issue #15190
```

Comment:
We don't usually include the fixes issue #, especially for an enhancement like this. Also, no need to mention adding tests, that is always the case!